### PR TITLE
feat: add create ledger wallet address by account index command

### PR DESCRIPTION
**Pull request by gnasnik**
## Checklist

Before you mark the PR ready for review, please make sure that:
- [ ] All commits have a clear commit message.
- [ ] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: cmd/lotus-shed/ledger.go [hunk 1]
```diff
@@ -291,3 +293,72 @@ var ledgerShowCmd = &cli.Command{
...
+	Name:  "new",
+	Flags: []cli.Flag{},
+	Action: func(cctx *cli.Context) error {
+		ctx := lcli.ReqContext(cctx)
+
+		if cctx.NArg() != 1 {
+			return fmt.Errorf("must pass account index")
+		}
+
+		index, err := strconv.Atoi(cctx.Args().First())
```
Comment (magik6k) [inline cmd/lotus-shed/ledger.go:307]: CodeQL probably wants you to use
```suggestion
		index, err := strconv.ParseUint(cctx.Args().First(), 10, 32)
```

**Review by magik6k — COMMENTED** [review]
Thanks for the PR!

Other than the CodeQL warning, this looks good; Will merge after that's addressed.

File: cmd/lotus-shed/ledger.go (hunk 1 above)
Comment (gnasnik) [reply cmd/lotus-shed/ledger.go]: done

**Review by magik6k — APPROVED** [review]
Thanks for the PR!

**Pull request merged**